server/controllers/schedule.js

```diff
@@ -0,0 +1,17 @@
+const Event = require("../models/Event");
+
+// @route POST /event/schedule
+// @desc search for entries in the database and returns the schedules
+// @access Private
+const schedule = async (req, res) => {
+    try{
+        const schedule = await Event.find( {userURL : req.params.userURL} );
+        const result = { 'schedule' : schedule};
+        res.json(result);
```

Comment:
We don't really need to create a new variable here, you could just pass that object directly to `res.json({ ... })`

Reply:
I will remove the new variable